File: C++/if_operadores.cpp
/* Uso de instrucciones if, operadores
    de relacion y operadores de igualdad */

#include <stdio.h>

int main() {
    int num1; /* Primer dato que lee el usuario */
    int num2; /* Segundo dato que lee el usuario */

    printf( "Introduzca dos numeros enteros y le dire\n");
    printf( "las relaciones que satisface: ");
    scanf("%d%d", &num1, &num2); /* Lectura de numeros enteros */

    if(num1 == num2) {
        printf("%d es igual que %d\n", num1, num2);
    } /* Fin de la instrucción if */

    if(num1 != num2) {
        printf("%d no es igual que %d\n", num1, num2);
    } /* Fin de la instrucción if */

    if(num1 < num2) {
        printf("%d es menor que %d\n", num1, num2);
    } /* Fin de la instrucción if */

    if(num1 > num2) {
        printf("%d es mayor que %d\n", num1, num2);
    } /* Fin de la instrucción if */

    if(num1 >= num2) {
        printf("%d es mayor o igual que %d\n", num1, num2);
    } /* Fin de la instrucción if */

    if(num1 <= num2) {
        printf("%d es menor o igual que %d\n", num1, num2);
    } /* Fin de la instrucción if */

    return 0;
} /* Fin de la función main */

File: Kotlin/if_operadores.kt
fun main() {
    var num1: Int //Al poner var significa que el valor es variable 
    var num2: Int

    println("Introduzca los valores y le dire que relaciones satisfacen:")
    num1 = readLine()!!.toInt() // Lee los numeros introducidos por el usuario y los convierte a enteros
    num2 = readLine()!!.toInt() // Lee los numeros introducidos por el usuario y los convierte a enteros

    if(num1 == num2) {
        println("$num1 es igual que $num2")
    } //FIn del if

    if(num1 != num2) {
        println("$num1 es diferente que $num2")
    }   //FIn del if

    if(num1 < num2) {
        println("$num1 es menor que $num2")
    } //FIn del if

    if(num1 > num2) {
        println("$num1 es mayor que $num2")
    } //FIn del if

    if(num1 <= num2) {
        println("$num1 es menor o igual que $num2")
    } //FIn del if

    if(num1 >= num2) {
        println("$num1 es mayor o igual que $num2")
    } //FIn del if
}